src/OpenTelemetry/OpenTelemetry.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net452;net46;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
After this PR we might be able to consolidate these `TargetFrameworks` (e.g. using the default setting in the top level `.props` file).